docs/resources/featurestudio.md

```diff
@@ -0,0 +1,234 @@
+## Syntax
+
+### Variables and Constants
+In Feature Studio, you can define a variable with `var name = value`. Later in code, when you use name, it will use the value associated with it. Additionally, if the value is assigned once and never changed, you can change the declaration to `const name = value`.
+
+### Functions
+A function is a block of code that is easily reusable and preforms a specific task. To define a function, you type `function myFunctionName()`. If you need to pass things into the function, you can use the syntax `function myFunctionName(var1 is Type)`. To execute the function, you can call it with the syntax `myFunctionName(var1)`
+
```

Comment:
A section on ValueWithUnits (when you need to multiply/divide by `inch`, how to print them to the console for debugging, etc.) would be nice. Ditto for Queries, I also have a video where I talk about them (and ids) in more detail you might find helpful as a reference
https://www.youtube.com/watch?v=JRxLxXwqWWQ